### DISCUSSION TITLE
2.0.16

### DISCUSSION THREAD

**Discussion by alapierre** [alapierre/ksef-java-rest-client#68] [Announcements]
dependencies update, map 400 response status to dedicated exception

<hr /><em>This discussion was created from the release <a href='https://github.com/alapierre/ksef-java-rest-client/releases/tag/2.0.16'>2.0.16</a>.</em>

**Comment by RobertBoksa** [comment]
Does the current version support the latest FA(2) schema?

**Reply by alapierre** [reply]
FA (2) is now supported in 2.0.18
